internal/service/user.go

```diff
@@ -24,56 +25,148 @@ func NewUserService(repo UserRepo, l logger.ZeroLog) UserService {
 }
 
 func (s UserService) Create(user entity.User) error {
-	// TODO: Implement me!
-	return nil
+	err := s.validateUser(user)
+	if err != nil {
+		return err
+	}
+	return s.repo.Create(user)
 }
 
 func (s UserService) Get(id uint64) (entity.User, error) {
-	// TODO: Implement me!
-	return entity.User{}, nil
+	user, err := s.repo.Read(id)
+	if err != nil {
+		return entity.User{}, err
+	}
+	return user, nil
```

Comment:
We could go straight forward and return from `s.repo.Read` function as the responses are the same. It allows reduce the code to a one-line function `return s.repo.Read(id)`